### DISCUSSION TITLE
License Clarification for Non-Lawyers

### DISCUSSION THREAD

**Discussion by TonyValenti** [SixLabors/ImageSharp#2647] [Q&A]
Hi @JimBobSquarePants and @brianpopow -
Can you help me out?

I use an open source library, [PdfSharpCore](https://github.com/ststeiger/PdfSharpCore/) which has a dependency on ImageSharp.
It is currently using the following super-old versions:
```
    <PackageReference Include="SixLabors.ImageSharp" Version="1.0.4" />
    <PackageReference Include="SixLabors.Fonts" Version="1.0.0-beta17" />
```

The maintainer feels that [he cannot upgrade the ImageSharp dependency for legal reasons](https://github.com/ststeiger/PdfSharpCore/issues/399#issuecomment-1779476827).

However, [the community feels differently](https://github.com/ststeiger/PdfSharpCore/issues/403#issuecomment-1817234239).

Could one of you please help the PdfSharp community out?  Here is what I'm hoping you can do:
1. Go to each of those two links above.
2. Create a post in which you: identify yourself as an author/maintainer of image sharp and clearly indicate that the package upgrade is allowed by the [SixLabors Split License](https://github.com/SixLabors/ImageSharp/blob/main/LICENSE).

Thank you so much for helping settle this legal discussion between non-lawyers.

**Comment by JimBobSquarePants** [comment]
I've commented but I doubt it will make much of a difference. Personally, I would either.

1. Try to advocate the upstream https://github.com/empira/PDFsharp library to take on the dependency.
2. Fork and maintain an updated version with others who are interested.

**Reply by TonyValenti** [reply]
Thank you!
